executor.py

```diff
@@ -41,13 +42,16 @@ def index(
     def search(
         self,
         docs: Optional['DocumentArray'] = None,
+        parameters: Optional[Dict] = None,
         **kwargs,
     ):
         """Perform a vector similarity search and retrieve the full Document match
 
         :param docs: the Documents to search with"""
```

Comment:
add docstr for `parameters`

Reply:
docstring for `parameters` is now added [here](https://github.com/jina-ai/executor-simpleindexer/pull/12/commits/9df6e2d235cc76a43b70bba6efbe0197d3c1c0b0).